pxl_scripts/px/redis_data/redis_data.pxl

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+''' Redis Data Tracer
+
+This script traces all Redis data on the cluster.
+'''
+
+import px
+
+
+def redis_data(start: str, num_head: int):
+
+    # redis_events.beta is a beta table, which is subject to change.
+    df = px.DataFrame(table='redis_events.beta', start_time=start)
+    df = add_source_dest_columns(df)
+
+    # Filter source column.
+    # df = df[df.source == 'default/cartservice']
```

Comment:
What do you think about leaving this so that it's obvious where to add additional filtering (before the `df.head()` and `add_source_dest_links()` calls)?

Reply:
I think it at least needs a comment explaining why it's there like, "you can add a filter here", but I think that it's still probably overkill

Reply:
I agree that hint is useful. In that case, we should add some script arguments for the source/destination filtering. That'll be useful across all scripts, and shows where filtering should be done.

And it can also be followed by a comment like: `# Add additional filters here`.

Reply:
- `redis_data`: I added `source_filter` and `destination_filter` which use px.contains() instead of exact string matching.
- `redis_flow_graph`: I renamed the to/from entity filtering to source/dest filtering. 
- `redis_stats` has a pod filter which checks that either the source or destination contains the filter name. It will filter IP addresses too.